library/src/main/java/com/cube/storm/ui/view/holder/list/ToggleableListItemViewHolder.java

```diff
@@ -38,6 +39,7 @@
 	protected TextView title;
 	protected TextView description;
 	protected LinearLayout embeddedLinksContainer;
+	private String titleText;
```

Comment:
This field can be scoped to the `populateView` function, rather than being class-scope